Makefile

```
@@ -184,9 +192,10 @@ UNDECOMPILED_DATA_DIRS := $(shell find data -type d)
 # source files
 C_FILES       := $(filter-out %.inc.c,$(foreach dir,$(SRC_DIRS) $(ASSET_BIN_DIRS),$(wildcard $(dir)/*.c)))
 S_FILES       := $(foreach dir,$(SRC_DIRS) $(UNDECOMPILED_DATA_DIRS),$(wildcard $(dir)/*.s))
+BIN_FILES     := $(wildcard baseroms/$(VERSION)/segments/*)
```

```suggestion
BASEROM_BIN_FILES := $(wildcard baseroms/$(VERSION)/segments/*)
```
rip the alignment but I think this is too generic of a name , in particular `$(foreach f,$(BIN_FILES),$(BUILD_DIR)/baserom/$(notdir $f).o)` later simply assumes it to be the baserom segments binaries